crispy_tailwind/__init__.py

```diff
@@ -1 +1 @@
-__version__ = "1.0.0"
+__version__ = "1.0.1"
```

Comment:
This isn't necessary and should be reverted because as of 143fe000419078546ddca772d547916d3208e305 this `__version__` string should be automatically incremented by AutoPub.

Reply:
👌